test/e2e-olm/utils.go

```diff
@@ -35,27 +37,47 @@ const (
 	exampleSecretsPath   = "/tmp/example-secrets"
 )
 
-func registerSchemes(t *testing.T) {
-	elasticsearchList := &loggingv1.ElasticsearchList{}
-	err := test.AddToFrameworkScheme(loggingv1.SchemeBuilder.AddToScheme, elasticsearchList)
+func setupK8sClient(t *testing.T) {
+	if _, ok := os.LookupEnv(TestOperatorNamespaceEnv); !ok {
+		t.Fatal("TEST_OPERATOR_NAMESPACE is unset")
+	}
+	operatorNamespace = os.Getenv(TestOperatorNamespaceEnv)
+	t.Logf("Found namespace: %q", operatorNamespace)
+
+	projectRootDir = getProjectRootPath("elasticsearch-operator")
+
+	var err error
+	cfg, err := config.GetConfig()
 	if err != nil {
```

Comment:
nit. No need for `var err error` here since you use `cfg, err := ...` on the next line.

Reply:
Fixed